loghub/main.py

```diff
@@ -14,6 +14,7 @@
 import datetime
 import sys
 import time
+import getpass
```

Comment:
@fpliger we use isort, so the error is due to this import not being correctly placed. If you run ciocheck it will get fixed utomatically